content/en/docs/getting-started/_index.md

```diff
@@ -103,12 +103,26 @@ sudo install -t /usr/local/bin {kubectl,oc}
 
 Copy the kubeconfig to the default location that can be accessed without administrator privilege.
 
+{{< tabs >}}
+{{% tab name="Podman" %}}
+
 ```Bash
 mkdir ~/.kube
 sudo podman cp microshift:/var/lib/microshift/resources/kubeadmin/kubeconfig ~/.kube/config
 sudo chown `whoami`: ~/.kube/config
 ```
 
+{{% /tab %}}
+{{% tab name=".rpm" %}}
+```Bash
+mkdir ~/.kube
+sudo cp /var/lib/microshift/resources/kubeadmin/kubeconfig ~/.kube/config
+sudo chown `whoami`: ~/.kube/config
```

Comment:
You can avoid the `chown` by just running:

```
sudo cat /var/lib/microshift/resources/kubeadmin/kubeconfig > ~/.kube/config
```

Reply:
patched! Thanks!